packages/vuetify/src/components/VTextField/VTextField.sass

```diff
@@ -433,10 +433,12 @@
 
     &.v-text-field--reverse legend
       +ltr()
-        text-align: right
+        margin-inline-start: auto
```

Comment:
IE doesn't support this property

Reply:
Thanks @jacekkarczmarczyk. Didn't knew about that, Update and tested on Chrome, Edge, IE and Firefox